debezium-reactive/src/main/java/io/debezium/reactive/internal/converters/ConverterRegistry.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.reactive.internal.converters;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ServiceLoader;
+
+import io.debezium.reactive.spi.AsType;
+import io.debezium.reactive.spi.Converter;
+
+/**
+ * A registry holding references to all converters available on classpath
+ *
+ * @author Jiri Pechanec
+ *
+ */
+public class ConverterRegistry {
+
+    private final Map<Class<? extends AsType<?>>, Converter<?>> registry = new HashMap<>();
```

Comment:
Would be good to make this unmodifiable.

Reply:
Converted to unmodifiableMap